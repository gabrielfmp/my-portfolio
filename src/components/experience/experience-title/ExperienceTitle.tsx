import { Dot, TitleWrapper } from './styles';
import { Title } from './../../../ui';
import React from 'react';

type ExperienceTitleProps = {
  title: string;
};

const ExperienceTitle: React.FC<ExperienceTitleProps> = ({ title }) => {
  return (
    <TitleWrapper>
      <Dot />
      <Title>{title}</Title>
    </TitleWrapper>
  );
};

export default ExperienceTitle;
