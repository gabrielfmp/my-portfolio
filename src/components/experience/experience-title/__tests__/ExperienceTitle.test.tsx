import { render } from '@testing-library/react';
import ExperienceTitle from '../ExperienceTitle';
import React from 'react';

test('snapshot', () => {
  const { asFragment } = render(
    <ExperienceTitle title="eu dou à música o que ela me dá" />
  );

  expect(asFragment()).toMatchSnapshot();
});
