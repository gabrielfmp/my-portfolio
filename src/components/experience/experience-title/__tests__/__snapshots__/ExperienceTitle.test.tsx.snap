// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-ov1ktg"
  >
    <div
      class="css-v5pni"
    />
    <h2
      class="css-111mxsp"
    >
      eu dou à música o que ela me dá
    </h2>
  </div>
</DocumentFragment>
`;
